src/realm.js

```diff
@@ -8,6 +8,7 @@ const { findDriver } = require('./drivers');
 const { camelCase } = require('./utils/string');
 const sequelize = require('./adapters/sequelize');
 const Raw = require('./raw');
+const { LEGACY_TIMESTAMP_MAP } = require('./contants');
```

Comment:
constants

didn't realize this typo until now :-(

Reply:
> constants
> 
> didn't realize this typo until now :-(

LOL, my left hand always quick than my right hand🙈